presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Enable spatial joins (broadcast and distributed) for ST_Equals, ST_Overlaps, ST_Crosses, and ST_Touches.
+* Fix a bug where index joins would fail with the error "driver should never block".  Now queries will wait until a configured timeout for the indexes to load.  The timeout can be configured with the configuration property ``index-loader-timeout`` and the session property ``index_loader_timeout``.
+* Move ``AggregationNode`` to SPI. Connectors can now push down aggregation to table scan.
+* Move ``ProjectNode`` to SPI. Connectors can now push down project to table scan.
+* Add ``task_partitioned_writer_count`` session property to allow to set number of concurrent writers for partitioned (bucketed) writes independently.
+* Removing syntax support for ``SET PATH``, ``CURRENT_PATH`` (Presto server do not use this information)
+* Fix NullPointerException in spatial join with geometry collections in the build side.
+* Add ``IPPREFIX`` type and :func:`ip_prefix` function.
+* Improve performance for queries with ``FULL OUTER JOIN`` where join keys have the :func:``COALESCE`` function applied.
```

Comment:
```
:func:`COALESCE`
```

Reply:
I don't know why but it doesn't work this way. 
```
:func:`coalesce` 
```
also doesn't work. The same 
```
:func:``COALESCE``
``` 
was also used in release-0.218.rst.

Reply:
This was not rendered properly in the 0.218 release notes:
https://prestodb.github.io/docs/current/release/release-0.218.html

You should just use double back quotes with out `:func:` prefix.

Reply:
ok used 
```
``COALESCE``
```
But it's still not resolvable.